### PR TITLE
Fix SEGV if app closed while toolbar customization was open

### PR DESCRIPTION
**Pull request by jhilmer**
Keep Toolbar alive until ToolbarAdapter is destroyed

### CODE REVIEW

File: src/core/gui/dialog/toolbarCustomize/ToolbarAdapter.cpp [hunk 1]
```diff
@@ -84,8 +84,10 @@ void ToolbarAdapter::cleanToolItem(GtkToolItem* it) {
     if (data) {
         gtk_widget_set_sensitive(GTK_WIDGET(it), ToolitemDragDrop::isToolItemEnabled(data));
     }
+    GdkWindow* window = gtk_widget_get_window(GTK_WIDGET(it));
 
-    gdk_window_set_cursor(gtk_widget_get_window(GTK_WIDGET(it)), nullptr);
+    if (window)
+        gdk_window_set_cursor(window, nullptr);
```
Comment (jhilmer) [inline src/core/gui/dialog/toolbarCustomize/ToolbarAdapter.cpp:90]: I receive `Gdk-CRITICAL **: 12:18:13.853: gdk_window_set_cursor: assertion 'GDK_IS_WINDOW (window)' failed` for the five GtkToolItem where have a submenu attached. Pen, Eraser, PdfTool, Draw, and Select. This hack remove this - but I don't like the solution. But for now the application can close without giving a SEGV.

Comment (rolandlo) [reply src/core/gui/dialog/toolbarCustomize/ToolbarAdapter.cpp:90]: So should we merge or are you still looking for a better solution? I can confirm that the segfault is fixed.

Comment (bhennion) [inline src/core/gui/dialog/toolbarCustomize/ToolbarAdapter.cpp:90]: Just one thing, to match our coding conventions:
```suggestion
    if (window) {
        gdk_window_set_cursor(window, nullptr);
    }
```

**Review by rolandlo — APPROVED** [review]
Squashing and merging in 24 hours if no objections are raised.

**Comment by rolandlo** [comment]
Thanks for the fix @jhilmer Just merged it.

**Pull request merged**